src/OrchardCore.Modules/Orchard.Users/Controllers/AccountController.cs

```diff
@@ -124,6 +124,28 @@ public IActionResult Register(string returnUrl = null)
             return Redirect("~/");
         }
 
+        [HttpGet]
+        public IActionResult ChangePassword()
+        {
+            return View();
+        }
+
+        [HttpPost]
+        [ValidateAntiForgeryToken]
+        public async Task<IActionResult> ChangePassword(ChangePasswordViewModel model)
+        {
+            if (ModelState.IsValid)
```

Comment:
The whole Account controller is marked with the [Authorize] attribute, so it shouldn't be neccessary